packages/bbui/src/ColorPicker/ColorPicker.svelte

```diff
@@ -124,20 +126,20 @@
     },
   ]
 
-  const getThemeClasses = theme => {
+  const getThemeClasses = (theme: Theme | undefined) => {
     if (!theme) {
       return ""
     }
     theme = ensureValidTheme(theme, DefaultAppTheme)
     return getThemeClassNames(theme)
   }
 
-  const onChange = value => {
+  const onChange = (value: string | null) => {
```

Comment:
I think we should be trying to get rid of `null` usage, as under typescript its very specific, `undefined` should work in all scenarios like this.

Reply:
I had this set to auto merge, woops. I will fix this in my next PR, sorry!